test-cli/src/bin/main.rs

```diff
@@ -98,21 +105,12 @@ fn main() {
 
             log::info!(&logger, "Generating {} SCIs...", num_quotes);
 
-            // We can't share our rng with the Rayon threads, but we still want a
-            // deterministic way to generate SCIs. This little hack allows us to
-            // do that by generating a unique seed for each SCI we will be generating.
-            let rng_seeds = (0..num_quotes)
+            let scis = (0..num_quotes)
                 .map(|_| {
-                    let mut seed_bytes = [0; 32];
-                    rng.fill_bytes(&mut seed_bytes);
-                    seed_bytes
-                })
-                .collect::<Vec<_>>();
-
-            let scis = rng_seeds
-                .into_par_iter()
-                .map(|rng_seed| {
-                    let mut rng: StdRng = SeedableRng::from_seed(rng_seed);
+                    // generate a unique seed for each SCI we will be generating.
+                    let mut rng_seed = [0; 32];
+                    rng.fill_bytes(&mut rng_seed);
+                    let mut reseeded_rng: StdRng = SeedableRng::from_seed(rng_seed);
```

Comment:
Why is this needed? The idea of the seed is that all SCIs end up generated deterministically and I believe that would happen if `rng` is used, without this stuff. Am I missing something?